go.mod

```diff
@@ -33,6 +33,7 @@ require (
 	github.com/linki/instrumented_http v0.2.0
 	github.com/linode/linodego v0.3.0
 	github.com/mattn/go-isatty v0.0.11 // indirect
+	github.com/maxatome/go-testdeep v1.4.0
```

Comment:
The changes are good, but I do have a question: is there a way to avoid adding a dependency just for testing?

Reply:
I don't think so :( This dependency is very important as all tests are using it. The error messages with it are very clear, and anything else would be either not readable or would require massive amount of code which would be practically duplicating what this dependency already does. Is it big issue?

Reply:
Afaik test dependencies don't affect production binary size itself